Lib/random.py

```diff
@@ -996,5 +996,75 @@ def _test(N=10_000):
     _os.register_at_fork(after_in_child=_inst.seed)
 
 
+# ------------------------------------------------------
+# -------------- command-line interface ----------------
+
+
+def parse_args(arg_list: list[str] | None):
```

Comment:
Might be better to make this private?

```suggestion
def _parse_args(arg_list: list[str] | None):
```